docs/content/get-started/installation/supported-platforms.md

```diff
@@ -14,7 +14,6 @@ Below are the platforms supported for the Aperture Agent:
 | Platform   | Extent of Support   |
 | ---------- | ------------------- |
 | Kubernetes | 1.23 and above      |
-| Istio      | 1.14 and above      |
```

Comment:
any reason why we removed it? @hdkshingala

Reply:
That support-platform was just for installation. We have version for istio under it's own page